jena-cmds/src/main/java/org/apache/jena/cmd/Cmds.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.jena.cmd;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Consumer;
+
+import org.apache.jena.sys.JenaSystem;
+
+/**
+ * Command execute by registering a name and a way to call "main".
+ * This is used by modules that are not a dependency of jena-cmds.
+ * <p>
+ * Example usage:
+ * <pre>
+ *
+ *
```

Comment:
WIP or missing example? (also not sure if it was "command execution" or commands executed", etc)

Reply:
Done